sdk/tests/py/lint/lint.whitelist

```diff
@@ -0,0 +1,40 @@
+# File containing whiteslist for lint errors
+# Format is:
+# ERROR TYPE:file/name/pattern[:line number]
+# e.g.
+# TRAILING WHITESPACE:example/file.html:128
+# to allow trailing whitespace on example/file.html line 128
+
+## Whitespace rules that we can't enforce yet ##
+
+TRAILING WHITESPACE:*.md
+TRAILING WHITESPACE:*.txt
+
+## Accept executable permission ##
```

Comment:
I'd suggest adding the following here since we want to minimize differences if these files are re-imported.

# Some of these imported files contain trailing whitespace.
TRAILING WHITESPACE:conformance/ogles/*.frag
TRAILING WHITESPACE:conformance/ogles/*.vert